cypress/e2e/tests/pages/manager/jwt-authentication.spec.ts

```diff
@@ -0,0 +1,151 @@
+
+import JWTAuthenticationPagePo from '@/cypress/e2e/po/pages/cluster-manager/jwt-authentication.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+
+describe('JWT Authentication', { testIsolation: 'off', tags: ['@manager', '@adminUser', '@jenkins'] }, () => {
+  const jwtAuthenticationPage = new JWTAuthenticationPagePo();
+
+  let instance0 = '';
+  let instance1 = '';
+  let removeCluster0 = false;
+  let removeCluster1 = false;
+  const region = 'us-west-1';
+  const namespace = 'fleet-default';
+
+  before(() => {
+    cy.login();
+    cy.createE2EResourceName('rke2cluster0').as('rke2Ec2ClusterName0');
+    cy.createE2EResourceName('rke2cluster1').as('rke2Ec2ClusterName1');
+
+    cy.get<string>('@rke2Ec2ClusterName0').then((name) => {
+      instance0 = name;
+      // create real cluster
+      cy.createAmazonRke2ClusterWithoutMachineConfig({
+        cloudCredentialsAmazon: {
+          workspace: namespace,
+          name,
+          region,
+          accessKey: Cypress.env('awsAccessKey'),
+          secretKey: Cypress.env('awsSecretKey')
+        },
+        rke2ClusterAmazon: {
+          clusterName: name,
+          namespace,
+        }
+      }).then(() => {
+        removeCluster0 = true;
+      });
+    });
+    cy.get<string>('@rke2Ec2ClusterName1').then((name) => {
+      instance1 = name;
+      // create real cluster
+      cy.createAmazonRke2ClusterWithoutMachineConfig({
+        cloudCredentialsAmazon: {
+          workspace: namespace,
+          name,
+          region,
+          accessKey: Cypress.env('awsAccessKey'),
+          secretKey: Cypress.env('awsSecretKey')
+        },
+        rke2ClusterAmazon: {
+          clusterName: name,
+          namespace,
+        }
+      }).then(() => {
+        removeCluster1 = true;
+      });
+    });
+  });
+  beforeEach(() => {
```

Comment:
this can be `before()` instead of `beforeEach()`. since `testIsolation` is off you don't need to login before each test.